spec.md

```diff
@@ -1784,6 +1785,27 @@ Condition | gRPC Code | Description | Recovery Behavior
 | --- | --- | --- | --- |
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
+#### `NodeDrain`
+
+A Node Plugin MUST implement this RPC call if the plugin has `DRAIN_VOLUMES` node capability.
+
+If the plugin has DRAIN_VOLUMES capability, the CO SHOULD call NodeDrain prior to terminating the plugin. During normal 
```

Comment:
Please review the contribution guidelines re: expected format for markdown text (e.g. one sentence per line), a single sentence shall not be split across multiple lines...

Also, this RPC is missing an error table that describes likely errors a CO might encounter and the expected CO behavior in such cases.